### ISSUE TITLE
The predicted results are not displayed on the Evaluate page.

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#1615]
**System Information (please complete the following information):**
- Windows OS: Windows-10-Enterprise-21H1
- ML.Net Model Builder (Preview): 16.6.3.2136604 (Main Branch)
- Microsoft Visual Studio Enterprise 2019: 16.10.2
- .Net: 5.0.301

**Describe the bug**
- On which step of the process did you run into an issue: Evaluate page
- Clear description of the problem: After clicking the "Predict" button on Evaluate page, the predicted results are not displayed in the Results area. 

**To Reproduce**
1. Select Create a new project from the Visual Studio 2019 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 5.0;
3. Add model builder by right click on the project;
4. Pass all steps to navigate to Evaluate page;
4. Click "Predict" button, the predicted results are not displayed after the spinner runs.

**Expected behavior**
The predicted results should be displayed after the spinner runs on Evaluate page.

**Screenshots**
![Evaluate issue](https://user-images.githubusercontent.com/81727020/126097345-f6bd440d-c0da-4029-b0d1-08f6b1cab71c.gif)

**Additional context**

**Comment by zewditu** [comment]
@vzhuqin  thanks for catching!

**Comment by vzhuqin** [comment]
Verified this issue on latest main build: 16.7.0.2137701
Encounter another issue for Text classification scenario, will verify again after #1648 is fixed.
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/e33004c1-e213-41af-9c05-bcf849845282)

Not repro this issue on other scenarios: Value prediction/Recommendation/Image classification/Object detection.
Value prediction:
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/4b4d3018-2611-4c9b-95af-d4c319e89498)
Recommendation：
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/88d8d742-55cf-4829-bcb6-15e9e6e6f441)
Image classification：
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/61037821-584d-4379-ba7c-10c9e9a4eaec)
Object detection：
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/0a9e28b2-e076-4742-82b5-01f341ec18fc)

**Comment by vzhuqin** [comment]
Not repro this issue on latest main build: 16.7.2.2140901
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/649ba7ec-cfc8-402e-87f3-05d7f66e6679)
